index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>setup and mobile first</title>
+</head>
+<body>
+    <header>
+        <a class="logo"> Dorcas</a>
+        <nav class="navlist">
+            <img src="img/Union.png" alt="navLogo">
+        </nav>
+    </header>
+
+    <main class="mainContent">
+        <section class="Content">
+            <h2>I'm Dorcas 
+                Glad to see you!
+            </h2>
+
+            <p>
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+
+            <div class="social media">
+                <h3>LET'S CONNECT</h3>
+                <ul>
+                    <li>
+                        <a href="#">
+                            <img src="img/Vector (7).png" alt="twitter image">
+                        </a>
+                    </li>
+                </ul>
+                <ul>
+                    <li>
+                        <a href="#">
+                            <img src="img/Vector (8).png" alt="linked image">
+                            <img src="img/Vector (9).png" alt="linked image">
+                        </a>
+                    </li>
+                </ul>
+                <ul>
+                    <li>
+                        <a href="#">
+                            <img src="img/Vector (10).png" alt="gmail image">
+                        </a>
+                    </li>
+                </ul>
+                <ul>
+                    <li>
+                        <a href="#">
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md). the list of icons uses a `ul` tag to group them, and a `li` per icon; each icon has a link (`a` tag) 

i.e all the list items should be grouped inside a single `ul` tag 🤓👍

Example

```HTML
 <ul>
      <li>
          <a href="#">
              <img src="img/Vector (10).png" alt="gmail image">
          </a>
      </li>
      <li>
          . . .
      </li>
      <li>
          . . .
      </li>
  </ul>
```

Kindly fix that 🛠💪